superset/views/database/views.py

```diff
@@ -102,10 +102,10 @@ def form_post(self, form):
 
         if not self.is_schema_allowed(database, schema_name):
             message = _(
-                'Database "{0}" Schema "{1}" is not allowed for csv uploads. '
-                "Please contact Superset Admin".format(
-                    database.database_name, schema_name
-                )
+                'Database "%(database_name)s" schema "%(schema_name)s" '
+                "is not allowed for csv uploads. Please contact Superset Admin",
```

Comment:
I think we're missing a possessive - should this be "Please contact your Superset admin"?

Reply:
FIxed that one and found another one at the same time. The other one had exclamation marks which I didn't change now; for uniformity it would be to decide on error grammar/formatting at some point.